app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :head do %>
+  <title>Details: <%= @task.name %></title>
+<% end %>
+
+<h1 class='title'>Details</h1>
+
+<main>
+  <p>
+    <%= render 'checkbox', { task: @task } %>
+    <span class='task_name'><%= @task.name %></span>
+
+    <span class='complete_date'>
+      <% if @task.completed_at %>
```

Comment:
Good to know! It's hard for me to judge what will make my app more "flexible" to future changes, and what is just making it complicated :)